lib/roll.rb

```diff
@@ -0,0 +1,7 @@
+class Roll
+  attr_reader :score
+
+  def initialize(score)
+    @score = score
+  end
+end
```

Comment:
see the comment on readme, I think splitting the rolls into a different class over-complicates it

Reply:
It came about only from trying to make the program modular.  Agreed it doesn't really suit this program though.